src/controller/ordersControllers.js

```diff
@@ -3,35 +3,35 @@ import orders from '../models/orders';
 
 
 export function postOrder (req, res) {
-    const {error} = validateOrders(req.body);
+	const {error} = validateOrders(req.body);
     
-    if(error) return res.status(400).json({status: 'success', message: error.details[0].message});
+	if(error) return res.status(400).json({status: 'Failed to create order', message: error.details[0].message});
```

Comment:
#### Nitpick
Please refactor this code and make it readable. It's obvious you aren't following the airbnb style guide.
For context, please check [this](https://github.com/airbnb/javascript#control-statements)

Reply:
I just implemented this, thanks